### DISCUSSION TITLE
Login fails when integrated with auth0.

### DISCUSSION THREAD

**Discussion by philosophia-dev** [Web3Auth/Web3Auth#1021] [Help]
Hello.
I am trying to integrate web3auth with auth0.
However, the same error continues to occur while trying to log in.
Of course, I set the callback URI of the application settings to https://beta.openlogin.com/auth, https://app.openlogin.com/auth on the auth0 dashboard, and I tried to follow the answer that depends on the similar case #886, but the problem remains.
Here I attach my error log and code.

![Screen Shot 2022-12-07 at 12 44 23 PM](https://user-images.githubusercontent.com/85674579/206084874-82270fdf-8b23-4108-b719-5b060b879579.jpg)
![Screen Shot 2022-12-07 at 1 00 54 PM](https://user-images.githubusercontent.com/85674579/206085563-43fab367-a401-4c06-bc26-8b40171ecdb2.jpg)

**Comment by shahbaz17** [comment]
Hey @sophiyuu 

Please update to the latest web3auth SDKs and try again. 

`package.json`
```json
    "@web3auth/base": "^4.0.0",
    "@web3auth/core": "^4.0.0",
    "@web3auth/openlogin-adapter": "^4.2.0",
```

For reference, I am sharing an [example app source code for Auth0 x Web3Auth](https://github.com/Web3Auth/examples/tree/main/web-core-sdk/custom-authentication/auth0-react-core-example).
